V1.65.4 migration fails

### Description

`V1.65.4__contract_result_hash.sql` fails for mirror node operators who have been running mirror nodes since 2019 before `transaction.transaction_hash` column was added so their db has it as null.

### Steps to reproduce

1. Set `transaction.transaction_hash` to `null` for a contract call transaction
2. Upgrade to v0.64.1

### Additional context

```bash
2022-09-14T11:04:09.798-0600 ERROR main o.s.b.SpringApplication Application run failed org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'flywayInitializer' defined in class path resource [org/springframework/boot/autoconfigure/flyway/FlywayAutoConfiguration$FlywayConfiguration.class]: Invocation of init method failed; nested exception is org.flywaydb.core.internal.command.DbMigrate$FlywayMigrateException: Migration V1.65.4__contract_result_hash.sql failed
--------------------------------------------------
SQL State  : 23502
Error Code : 0
Message    : ERROR: column "transaction_hash" of relation "contract_result" contains null values
Location   : db/migration/v1/V1.65.4__contract_result_hash.sql (/Users/stevensheehy/projects/hedera/hedera-mirror-node/hedera-mirror-importer/target/file:/Users/stevensheehy/projects/hedera/hedera-mirror-node/hedera-mirror-importer/target/hedera-mirror-importer-0.64.1.jar!/BOOT-INF/classes!/db/migration/v1/V1.65.4__contract_result_hash.sql)
Line       : 22
Statement  : alter table if exists contract_result
    alter column transaction_hash set not null,
    alter column transaction_result set not null

	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.initializeBean(AbstractAutowireCapableBeanFactory.java:1804)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:620)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:542)
	at org.springframework.beans.factory.support.AbstractBeanFactory.lambda$doGetBean$0(AbstractBeanFactory.java:335)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:234)
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:333)
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:208)
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:322)
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:208)
	at org.springframework.context.support.AbstractApplicationContext.getBean(AbstractApplicationContext.java:1154)
	at org.springframework.context.support.AbstractApplicationContext.finishBeanFactoryInitialization(AbstractApplicationContext.java:908)
	at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:583)
	at org.springframework.boot.web.reactive.context.ReactiveWebServerApplicationContext.refresh(ReactiveWebServerApplicationContext.java:66)
	at org.springframework.boot.SpringApplication.refresh(SpringApplication.java:734)
	at org.springframework.boot.SpringApplication.refreshContext(SpringApplication.java:408)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:308)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1306)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1295)
	at com.hedera.mirror.importer.MirrorImporterApplication.main(MirrorImporterApplication.java:33)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.springframework.boot.loader.MainMethodRunner.run(MainMethodRunner.java:49)
	at org.springframework.boot.loader.Launcher.launch(Launcher.java:108)
	at org.springframework.boot.loader.Launcher.launch(Launcher.java:58)
	at org.springframework.boot.loader.JarLauncher.main(JarLauncher.java:65)
Caused by: org.flywaydb.core.internal.command.DbMigrate$FlywayMigrateException: Migration V1.65.4__contract_result_hash.sql failed
--------------------------------------------------
SQL State  : 23502
Error Code : 0
Message    : ERROR: column "transaction_hash" of relation "contract_result" contains null values
Location   : db/migration/v1/V1.65.4__contract_result_hash.sql (/Users/stevensheehy/projects/hedera/hedera-mirror-node/hedera-mirror-importer/target/file:/Users/stevensheehy/projects/hedera/hedera-mirror-node/hedera-mirror-importer/target/hedera-mirror-importer-0.64.1.jar!/BOOT-INF/classes!/db/migration/v1/V1.65.4__contract_result_hash.sql)
Line       : 22
Statement  : alter table if exists contract_result
    alter column transaction_hash set not null,
    alter column transaction_result set not null

	at org.flywaydb.core.internal.command.DbMigrate.doMigrateGroup(DbMigrate.java:385)
	at org.flywaydb.core.internal.command.DbMigrate.lambda$applyMigrations$1(DbMigrate.java:275)
	at org.flywaydb.core.internal.jdbc.TransactionalExecutionTemplate.execute(TransactionalExecutionTemplate.java:55)
	at org.flywaydb.core.internal.command.DbMigrate.applyMigrations(DbMigrate.java:274)
	at org.flywaydb.core.internal.command.DbMigrate.migrateGroup(DbMigrate.java:247)
	at org.flywaydb.core.internal.command.DbMigrate.lambda$migrateAll$0(DbMigrate.java:141)
	at org.flywaydb.core.internal.database.postgresql.PostgreSQLAdvisoryLockTemplate.execute(PostgreSQLAdvisoryLockTemplate.java:69)
	at org.flywaydb.core.internal.database.postgresql.PostgreSQLConnection.lock(PostgreSQLConnection.java:99)
	at org.flywaydb.core.internal.schemahistory.JdbcTableSchemaHistory.lock(JdbcTableSchemaHistory.java:139)
	at org.flywaydb.core.internal.command.DbMigrate.migrateAll(DbMigrate.java:141)
	at org.flywaydb.core.internal.command.DbMigrate.migrate(DbMigrate.java:98)
	at org.flywaydb.core.Flyway$1.execute(Flyway.java:172)
	at org.flywaydb.core.Flyway$1.execute(Flyway.java:124)
	at org.flywaydb.core.FlywayExecutor.execute(FlywayExecutor.java:205)
	at org.flywaydb.core.Flyway.migrate(Flyway.java:124)
	at org.springframework.boot.autoconfigure.flyway.FlywayMigrationInitializer.afterPropertiesSet(FlywayMigrationInitializer.java:66)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.invokeInitMethods(AbstractAutowireCapableBeanFactory.java:1863)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.initializeBean(AbstractAutowireCapableBeanFactory.java:1800)
	... 26 more
Caused by: org.flywaydb.core.internal.sqlscript.FlywaySqlScriptException: Migration V1.65.4__contract_result_hash.sql failed
--------------------------------------------------
SQL State  : 23502
Error Code : 0
Message    : ERROR: column "transaction_hash" of relation "contract_result" contains null values
Location   : db/migration/v1/V1.65.4__contract_result_hash.sql (/Users/stevensheehy/projects/hedera/hedera-mirror-node/hedera-mirror-importer/target/file:/Users/stevensheehy/projects/hedera/hedera-mirror-node/hedera-mirror-importer/target/hedera-mirror-importer-0.64.1.jar!/BOOT-INF/classes!/db/migration/v1/V1.65.4__contract_result_hash.sql)
Line       : 22
Statement  : alter table if exists contract_result
    alter column transaction_hash set not null,
    alter column transaction_result set not null

	at org.flywaydb.core.internal.sqlscript.DefaultSqlScriptExecutor.handleException(DefaultSqlScriptExecutor.java:275)
	at org.flywaydb.core.internal.sqlscript.DefaultSqlScriptExecutor.executeStatement(DefaultSqlScriptExecutor.java:222)
	at org.flywaydb.core.internal.sqlscript.DefaultSqlScriptExecutor.execute(DefaultSqlScriptExecutor.java:126)
	at org.flywaydb.core.internal.resolver.sql.SqlMigrationExecutor.executeOnce(SqlMigrationExecutor.java:69)
	at org.flywaydb.core.internal.resolver.sql.SqlMigrationExecutor.lambda$execute$0(SqlMigrationExecutor.java:58)
	at org.flywaydb.core.internal.database.DefaultExecutionStrategy.execute(DefaultExecutionStrategy.java:27)
	at org.flywaydb.core.internal.resolver.sql.SqlMigrationExecutor.execute(SqlMigrationExecutor.java:57)
at org.flywaydb.core.internal.command.DbMigrate.doMigrateGroup(DbMigrate.java:377)
	... 43 more
Caused by: org.postgresql.util.PSQLException: ERROR: column "transaction_hash" of relation "contract_result" contains null values
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2676)
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:2366)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:356)
	at org.postgresql.jdbc.PgStatement.executeInternal(PgStatement.java:490)
	at org.postgresql.jdbc.PgStatement.execute(PgStatement.java:408)
	at org.postgresql.jdbc.PgStatement.executeWithFlags(PgStatement.java:329)
	at org.postgresql.jdbc.PgStatement.executeCachedSql(PgStatement.java:315)
	at org.postgresql.jdbc.PgStatement.executeWithFlags(PgStatement.java:291)
	at org.postgresql.jdbc.PgStatement.execute(PgStatement.java:286)
	at org.flywaydb.core.internal.jdbc.JdbcTemplate.executeStatement(JdbcTemplate.java:201)
	at org.flywaydb.core.internal.sqlscript.ParsedSqlStatement.execute(ParsedSqlStatement.java:95)
	at org.flywaydb.core.internal.sqlscript.DefaultSqlScriptExecutor.executeStatement(DefaultSqlScriptExecutor.java:210)
	... 49 more
```

### Hedera network

other

### Version

v0.64.1

### Operating system

_No response_

--- a/hedera-mirror-importer/src/main/resources/db/migration/v1/V1.65.4.1__contract_result_hash.sql
+++ b/hedera-mirror-importer/src/main/resources/db/migration/v1/V1.65.4.1__contract_result_hash.sql
@@ -20,5 +20,4 @@ from results r
 where r.consensus_timestamp = contract_result.consensus_timestamp;
 
 alter table if exists contract_result
-    alter column transaction_hash set not null,
     alter column transaction_result set not null;

